report/renderer/report-ui-features.js

```diff
@@ -324,25 +324,12 @@ export class ReportUIFeatures {
   }
 
   /**
-   * Downloads a file (blob) using a[download].
-   * @param {Blob|File} blob The file to save.
+   * Wrapper is redefined and used by DevTools.
```

Comment:
Doesn't everything still use this, not just DT?

Reply:
I think the reports could just call the new function on `dom` directly, but DT hooks into calls to `ReportUIFeatuers._saveFile` so it was causing errors in DT tests.

https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/panels/lighthouse/LighthouseReportRenderer.ts;l=183-197?q=ReportUIFeatures&ss=chromium%2Fchromium%2Fsrc:third_party%2Fdevtools-frontend%2Fsrc%2F

Reply:
Comment could be better though ya, will update